src/main/java/org/jenkinsci/plugins/workflow/multibranch/AbstractWorkflowMultiBranchProjectFactory.java

```
@@ -56,7 +56,7 @@
         } // otherwise got recognized by something else before, oh well
     }
 
-    protected void customize(WorkflowMultiBranchProject project) {}
+    protected void customize(WorkflowMultiBranchProject project) throws IOException {}
```

May as well let it throw `InterruptedException` too.